### ISSUE TITLE
GitHub pages tunning styles

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#142]
Make GitHub pages nicer and probably split them into the multiple pages.

**Comment by TomazErjavec** [comment]
Yes, this is in fact a long-standing goal, now they are extremely boring. A much more polished example is [TEI Lex0](https://dariah-eric.github.io/lexicalresources/pages/TEILex0/TEILex0.html) and I guess stuff could be snatches from there.
But I guess this won't be done for 4.1...

Btw. I did try splitting the pages, but got pretty weird results.
